README.md

```diff
@@ -403,7 +403,7 @@ $ curl localhost:1338/latest/meta-data/mac
       <title>401 - Unauthorized</title>
    </head>
    <body>
-      <h1>401 - Unauthorized</h1>
+      <h1.5.0 - Unauthorized</h1>
```

Comment:
should be the last time since we're outta the 4s now! Deff just as good as a perma fix